README.md

```diff
@@ -27,34 +27,40 @@ Clients establish a websocket connection to push configurable telemetry records.
   "namespace": string - kafka topic prefix,
   "reliable_ack": bool - for use with reliable datastores, recommend setting to true with kafka,
   "monitoring": {
-      "prometheus_metrics_port": int,
-      "profiler_port": int,
-      "profiling_path": string - out path,
-      "statsd": { if you are not using prometheus
-        "host": string - host:port of the statsd server,
-        "prefix": string - prefix for statsd metrics,
-        "sample_rate": int - 0 to 100 percentage to sample stats,
-        "flush_period": int - ms flush period
-      }
+    "prometheus_metrics_port": int,
+    "profiler_port": int,
+    "profiling_path": string - out path,
+    "statsd": { if you are not using prometheus
+      "host": string - host:port of the statsd server,
+      "prefix": string - prefix for statsd metrics,
+      "sample_rate": int - 0 to 100 percentage to sample stats,
+      "flush_period": int - ms flush period
+    }
   },
+  "kinesis" {
+    "max_retries": 3,
+    "streams": {
+      "V": "custom_stream_name"
```

Comment:
what's V?

Reply:
That's the default vehicle data stream.  There are currently 3: `V`, `errors`, `alerts`.  Single letter for vehicle data because it's high volume.

Reply:
Could we spell vehicle_data out in configuration and map it to "V" during initial config setup? It isn't the most intuitive in the configuration file.

Reply:
Got it, make sense! Is this documented somewhere?

Reply:
Updated the readme to make that more clear